### PR TITLE
feat: add more explanation in the "How to Use" section #149

### PR DESCRIPTION
**Pull request by salonigoyal2309**
Issue: #149
If you want any other changes please let me know :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,17 @@ In case of using buttons that have icons in them, make sure to include font awes
 ```
 <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.14.0/css/all.css" integrity="sha384-HzLeBuhoNPvSl5KYnjx0BT+WB0QEEqLprO+NBkkk5gbc67FTaL7XIGa2w1L0Xbgc" crossorigin="anonymous">
 ```
+For using sButtons in your code , just add the classes of sButton you want to either  ```<button>``` or ```<a>``` tags :
```
Comment (shahednasser) [inline README.md:28]: Please replace "For using sButtons in your code" with "To use sButtons in your project. (Make the change in `index.html` as well

File: README.md [hunk 2]
```diff
@@ -25,6 +25,17 @@ In case of using buttons that have icons in them, make sure to include font awes
...
+
+```
+<button class='sbtn basic-btn blue-btn'> text </button>
```
Comment (shahednasser) [inline README.md:31]: remove the spacing around "text" and replace "text" with "Button" (Make this change on `index.html` as well)

File: README.md [hunk 3]
```diff
@@ -25,6 +25,17 @@ In case of using buttons that have icons in them, make sure to include font awes
...
+```
+This code will produce basic sButton with blue colour , same can be done for ```<a>``` tag
+
+```
+<a href='#' class='sbtn basic-btn blue-btn'> text </a>
+```
+You can find all classes and their corresponding sButtons mentioned in our website.
```
Comment (shahednasser) [inline README.md:38]: Add a link to the website at "our website"

File: README.md [hunk 4]
```diff
@@ -25,6 +25,17 @@ In case of using buttons that have icons in them, make sure to include font awes
 ```
 <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.14.0/css/all.css" integrity="sha384-HzLeBuhoNPvSl5KYnjx0BT+WB0QEEqLprO+NBkkk5gbc67FTaL7XIGa2w1L0Xbgc" crossorigin="anonymous">
 ```
+For using sButtons in your code , just add the classes of sButton you want to either  ```<button>``` or ```<a>``` tags :
+
+```
+<button class='sbtn basic-btn blue-btn'> text </button>
+```
+This code will produce basic sButton with blue colour , same can be done for ```<a>``` tag
```
Comment (shahednasser) [inline README.md:33]: after "blue colour" put a period (.) instead of comma (same for `index.html`)

**Review by shahednasser — CHANGES_REQUESTED** [review]

**Comment by salonigoyal2309** [comment]
@shahednasser this pull request is now having conflict with original repo , so what should I do ?

File: README.md [hunk 5]
```diff
@@ -25,17 +25,17 @@ In case of using buttons that have icons in them, make sure to include font awes
...
 ```
-<button class='sbtn basic-btn blue-btn'> text </button>
+<button class='sbtn basic-btn blue-btn'>Button</button>
 ```
-This code will produce basic sButton with blue colour , same can be done for ```<a>``` tag
+This code will produce basic sButton with blue colour. Same can be done for ```<a>``` tag
 
 ```
-<a href='#' class='sbtn basic-btn blue-btn'> text </a>
+<a href='#' class='sbtn basic-btn blue-btn'>text</a>
```
Comment (shahednasser) [inline README.md:36]: Please change this to Button too. (same for index.html)

**Review by shahednasser — CHANGES_REQUESTED** [review]
Don't worry about the conflict. We'll resolve it once you make this last change!

**Comment by shahednasser** [comment]
Great thank you!

@all-contributors please add @salonigoyal2309 for code and documentation.

**Pull request merged**